roles/maintenance_10_linux/vars/Debian.yml

```
@@ -1,3 +1,4 @@
 ---
 
 linux_allowed_login_since: -90days
+logrotate_status_file: "/var/lib/logrotate/status"
```

Please prefix the vars with the role name:
```suggestion
linux_logrotate_status_file: "/var/lib/logrotate/status"
```